blocks/api/serializer.js

```diff
@@ -55,7 +56,7 @@ export function getSaveContent( blockType, attributes ) {
 			return element;
 		}
 
-		const extraProps = {};
+		const extraProps = applyFilters( 'getSaveContent.extraProps', {}, blockType, attributes );
```

Comment:
This one seems to be a very specific. Maybe it should be simply called `addAdvancedAttributes` instead of `getSaveContent.extraProps` to avoid dot notation in the name.

Reply:
Actually, I think `getSaveContent.extraProps` convey the correct meaning. I don't care much about the dot notation